src/app/onion/onion.module.ts

```diff
@@ -58,7 +57,8 @@ import { DashboardComponent } from './dashboard/dashboard.component';
     LearningObjectBuilderModule,
     OnionRoutingModule,
     VirtualScrollerModule,
-    SidePanelModule
+    SidePanelModule,
+    DashboardModule
```

Comment:
Trailing commas will prevent unneeded git diff as seen here.
```suggestion
    DashboardModule,
```